charts/incubator/trilium-notes/values.yaml

```diff
@@ -0,0 +1,16 @@
+image:
+  repository: hub.docker.com/r/zadam/trilium
+  pullPolicy: IfNotPresent
+  tag: v0.49.3@sha256:01305359c92238d1bd892f65ff035635d59359bb3b91919b473240937560f529
```

Comment:
file a PR to the container repository and add the mirror if you can.
if not i'll add it there in a few days or so.

Reply:
Changed from docker hub to ghcr.io.

Reply:
Related: https://github.com/truecharts/containers/pull/89